ansible/roles/riakcs/sync_keys/tasks/main.yml

```diff
@@ -15,3 +15,21 @@
     state: present
   notify: "restart {{ name }}"
   when: "'riak_key' in hostvars[riakcs_control] and 'riak_secret' in hostvars[riakcs_control]"
+
+- name: Update access_id in riakcs datadog agent config
+  lineinfile:
+    dest: "/etc/dd-agent/conf.d/riakcs.yaml"
+    regexp: "([ -]+access_id: ).*$"
+    line: "\\1{{ hostvars[riakcs_control].get('riak_key', 'not-set') }}"
+    backrefs: yes
+  when: "name == 'riakcs' and 'riak_key' in hostvars[riakcs_control] and 'riak_secret' in hostvars[riakcs_control]"
+  notify: restart datadog
+
+- name: Update access_secret in riakcs datadog agent config
+  lineinfile:
+    dest: "/etc/dd-agent/conf.d/riakcs.yaml"
+    regexp: "([ -]+ access_secret: ).*$"
+    line: "\\1{{ hostvars[riakcs_control].get('riak_secret', 'not-set') }}"
+    backrefs: yes
+  when: "name == 'riakcs' and 'riak_key' in hostvars[riakcs_control] and 'riak_secret' in hostvars[riakcs_control]"
+  notify: restart datadog
```

Comment:
These new tasks mean that datadog must be installed before riakcs can be installed. Does that seem like a reasonable dependency? Should I make these tasks conditional on whether datadog is present or not? Actually, I think the dependency should be on the presence of the `datadog-agent` service and possibly `/etc/dd-agent/conf.d`. Thoughts welcome.